### PR TITLE
EWS and ILS migration changes

### PR DESCRIPTION
**Pull request by kkrampa**
@czue I think that PR: #5092  became a little bit messy so I make cleaner version of it. You should be able to read this PR commit by commit. If it's ok for you I will close #5092.

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
i'm not sure when i'll be able to review this since i'm traveling for the next week - @twymer feel free to take a first pass and merge if you're satisfied and i haven't been able to look

File: custom/logistics/api.py [hunk 1]
```diff
@@ -75,3 +84,131 @@ def get_stocktransactions(self, **kwargs):
...
+            except Exception as e:
+                logging.error(e)
+        else:
+            if self.domain not in user.get_domains():
+                user.add_domain_membership(self.domain, location_id=location_id)
+                user.save()
+        return user
+
+    def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
+                       first_name='', last_name=''):
```
Comment (dimagimon) [inline custom/logistics/api.py]: E127 continuation line over-indented for visual indent

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by kkrampa** [comment]
@twymer I made answers to your questions. If you have some comments that don't prevent from merge, please make them and I will fix them in another PR.

**Pull request merged**